cli/src/main.rs

```diff
@@ -0,0 +1,41 @@
+use std::str::FromStr;
+
+use clap::{command, Parser, Subcommand};
+
+#[derive(Parser, Debug)]
+#[command(author, version, about, long_about = None)]
+#[command(propagate_version = true)]
+struct CliArguments {
+    #[command(subcommand)]
+    command: Command,
+}
+
+#[derive(Subcommand, Debug)]
+enum Command {
+    TestIssueCredential {
+        #[arg(short, long)]
+        input_file_path: String,
+    },
+    TestIssuePresentation {
+        #[arg(short, long)]
+        input_file_path: String,
+    },
+}
+
+fn main() {
+    let args = CliArguments::parse();
+    match args.command {
+        Command::TestIssueCredential { input_file_path } => {
+            //read the JSON string file in input_file_path and parse it into a VerifiableCredential
+            //print the VerifiableCredential
```

Comment:
`clap` will let you autogenerate command line help text from doc comments (see the example on https://docs.rs/clap/latest/clap/) - maybe this comment could live on the definition of `TestIssueCredential` as a doc comment.

Reply:
That's great to know! Let me see

Reply:
Done